### ISSUE TITLE
Release - June 2018 - name: 0.15.1

### ISSUE THREAD

**Issue by laurentlb** [bazelbuild/bazel#5591]
Path release on top of release 0.15.0 (https://github.com/bazelbuild/bazel/issues/5057)

**Comment by laurentlb** [comment]
Need to cherry-pick 8ff87c164f48dbabe3b20becd00dde90c50d46f5 (to fix https://github.com/bazelbuild/bazel/issues/5468)

Anything else?

**Comment by ittaiz** [comment]
Weren’t they two commits to fix 5468?

**Comment by mhlopko** [comment]
Yes pleease, https://github.com/bazelbuild/bazel/commit/539cdca24c65e6b93503ffdce940bbb0ee236232 and https://github.com/bazelbuild/bazel/commit/c25f7c83fc3c327df3a2e6a305fdb7ba19788737 for #5468

**Comment by mhlopko** [comment]
Sorry those are from the 0.16 release branch.

https://github.com/bazelbuild/bazel/commit/8ff87c164f48dbabe3b20becd00dde90c50d46f5 and
https://github.com/bazelbuild/bazel/commit/c4622ac9205d2f1b42dac8c598e83113d39e7f11 from the master branch.

**Comment by buchgr** [comment]
please cherry-pick d3228b61f633cdc5b3f740b641a0836f1bd79abd

**Comment by laurentlb** [comment]
Cherry-picked the 3 commits mentioned above.
https://releases.bazel.build/0.15.1/rc1/index.html

**Comment by iabraham** [comment]
Any timeline on when the patch to fix #5468 will make it to Homebrew?

**Comment by ittaiz** [comment]
@laurentlb @buchgr any info on this making it to Homebrew?

**Comment by laurentlb** [comment]
I expect to release 0.15.1 later today.
Homebrew can be updated after that.

**Comment by ittaiz** [comment]
Oh, I thought it was released already. Thanks!
🤞

**Comment by vbatts** [comment]
https://copr.fedorainfracloud.org/coprs/vbatts/bazel/build/777798/ submitted

**Comment by ilovezfs** [comment]
0.15.1 shipped in Homebrew: https://github.com/Homebrew/homebrew-core/pull/30120

**Comment by petemounce** [comment]
chocolatey 0.15.1 is also pushed.

**Comment by schroederc** [comment]
Running `bazel-0.15.1-linux-x86_64` gives me this error:

```
Extracting Bazel installation...
FATAL: Could not find system javabase. Ensure JAVA_HOME is set, or javac is on your PATH.
```

`bazel-0.15.1-linux-x86_64` seems to now require either `JAVA_HOME` set or `javac` on the system's `PATH` where as `bazel-0.15.0-linux-x86_64` did not.  I'm assuming previous versions simply used the embedded JDK and now Bazel does not, but the release notes do not specify any such change.

**Comment by buchgr** [comment]
@schroederc thanks for making us aware. in fact this binary was created without an embedded jdk. that's a bug. I am looking into it.

**Comment by ittaiz** [comment]
Jakob,
Do you know if this is on OS X as well?

**Comment by buchgr** [comment]
@ittaiz yep!

We are currently releasing 0.15.2. Same cherry-picks, but built with the embedded jdk.

**Comment by laurentlb** [comment]
I've released 0.15.2: https://github.com/bazelbuild/bazel/releases/tag/0.15.2
It is what 0.15.1 should have been (same set of cherrypicks; without the jdk issue).

Sorry for the issue.

cc @ilovezfs @vbatts @petemounce

**Comment by vbatts** [comment]
fedora/centos build kicked off https://copr.fedorainfracloud.org/coprs/vbatts/bazel/build/778179/

**Comment by ilovezfs** [comment]
0.15.2 shipped in Homebrew https://github.com/Homebrew/homebrew-core/pull/30184

**Comment by ahedberg** [comment]
Hi folks - Chocolatey on Windows is suggesting 0.15.1 as the newest version, and Abseil started seeing the following error on its internal CI when it was used:

bazel clean 
Extracting Bazel installation...
ERROR: Couldn't find java at 'C:\Program Files\/bin/java.exe'.

We're pinning to 0.15.0 in the meantime. Sounds like 0.15.2 might fix this, but thought I'd mention it in case someone has contacts with Chocolatey.

**Comment by petemounce** [comment]
I've just pushed 0.15.2 to chocolatey.
